dap/src/types.rs

```diff
@@ -372,6 +376,7 @@ pub enum PresentationHint {
 }
 
 #[derive(Deserialize, Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
 #[cfg_attr(feature = "integration_testing", derive(Dummy))]
 pub struct Checksum {
```

Comment:
In a small subset of cases, there was a missing `camelCase` that wasn't strictly necessary. I added them in anyways for completeness, since it seemed like you were adding them elsewhere even if not strictly required